File: SDKs/Aspose.Words_Cloud_SDK_for_Java/src/main/java/com/aspose/words/model/Hyperlink.java
package com.aspose.words.model;

import com.aspose.words.model.Link;
public class Hyperlink {
  private String DisplayText = null;
  private String Value = null;
  private Link link = null;
  /**
	 * getDisplayText
	 * Gets String
	 * @return DisplayText
	 */
  public String getDisplayText() {
    return DisplayText;
  }

	/**
	 * setDisplayText
	 * Sets String
	 * @param DisplayText String
	 */
  public void setDisplayText(String DisplayText) {
    this.DisplayText = DisplayText;
  }

  /**
	 * getValue
	 * Gets String
	 * @return Value
	 */
  public String getValue() {
    return Value;
  }

	/**
	 * setValue
	 * Sets String
	 * @param Value String
	 */
  public void setValue(String Value) {
    this.Value = Value;
  }

  /**
	 * getLink
	 * Gets Link
	 * @return link
	 */
  public Link getLink() {
    return link;
  }

	/**
	 * setLink
	 * Sets Link
	 * @param link Link
	 */
  public void setLink(Link link) {
    this.link = link;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Hyperlink {\n");
    sb.append("  DisplayText: ").append(DisplayText).append("\n");
    sb.append("  Value: ").append(Value).append("\n");
    sb.append("  link: ").append(link).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Words_Cloud_SDK_for_.NET/src/Com/Aspose/Words/Model/Hyperlink.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Words.Model {
  public class Hyperlink {
    public string DisplayText { get; set; }

    public string Value { get; set; }

    public Link link { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Hyperlink {\n");
      sb.Append("  DisplayText: ").Append(DisplayText).Append("\n");
      sb.Append("  Value: ").Append(Value).Append("\n");
      sb.Append("  link: ").Append(link).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
